request-postgres/Dockerfile

```
@@ -0,0 +1,36 @@
+FROM debian:12-slim
+
```

For security, consider running as a user
```suggestion
USER 1001
```

---

Not sure what this would achieve?

---

https://docs.docker.com/develop/develop-images/instructions/#user
> If a service can run without privileges, use USER to change to a non-root user.

It's a security best practice. This same comment applies to all the docker images.

---

Decided to leave "as is" due to all other docker images